docs/coordinates/velocities.rst

```diff
@@ -417,8 +417,11 @@ radial velocity of 10 km/s:
                radial_velocity=[10]*1000*u.km/u.s,
                location=location, obstime=time)
     gcrs = aa.transform_to(GCRS(obstime=time))
-    plt.plot_date(time.plot_date, gcrs.radial_velocity.to(u.km/u.s))
-    plt.ylabel('RV [km/s]')
+
+    fig, ax = plt.subplots()
+    ax.xaxis.axis_date()
+    ax.scatter(time.plot_date, gcrs.radial_velocity.to(u.km/u.s))
+    ax.set_ylabel('RV [km/s]')
```

Comment:
You are editing the same lines in #17316. Which order should the pull requests be reviewed in?

Reply:
Order doesn't matter, I'll need to handle a conflict in all cases.